ci-scripts/setup.sh

```diff
@@ -5,3 +5,17 @@ set -o errexit
 set -o pipefail
 
 echo "TODO: implement installation and setup of backstage to given openshift cluster"
+
+export GITHUB_TOKEN QUAY_TOKEN KUBECONFIG
+
+GITHUB_TOKEN=$(cat /usr/local/ci-secrets/backstage-performance/github.token)
+QUAY_TOKEN=$(cat /usr/local/ci-secrets/backstage-performance/quay.token)
+
+
+rm -rvf .rhdh-setup.git*
+
+cd $(mktemp -d .rhdh-setup.git-XXX)
+
+git clone --branch main git@github.com:yogananth-subramanian/rhdh-setup.git .
```

Comment:
This will be eventually incorporated into this repo... but for now I just consume @yogananth-subramanian's repo